casepro/msg_board/models.py

```diff
@@ -0,0 +1,11 @@
+from dash.orgs.models import Org
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class PinnedComment(models.Model):
```

Comment:
I think it would be cleaner to add a `is_pinned` field to a custom comment model (https://django-contrib-comments.readthedocs.io/en/latest/custom.html) rather than have two different models